connect/transforms/src/main/java/org/apache/kafka/connect/transforms/ExtractField.java

```diff
@@ -24,13 +24,14 @@
 
 import java.util.Map;
 
-import static org.apache.kafka.connect.transforms.util.Requirements.requireMap;
-import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
+import static org.apache.kafka.connect.transforms.util.Requirements.requireMapOrNull;
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStructOrNull;
 
 public abstract class ExtractField<R extends ConnectRecord<R>> implements Transformation<R> {
 
     public static final String OVERVIEW_DOC =
-            "Extract the specified field from a Struct when schema present, or a Map in the case of schemaless data."
+            "Extract the specified field from a Struct when schema present, or a Map in the case of schemaless data. "
+                    + "null values are passed through unmodified."
```

Comment:
I think that's the optimal compromise. Keep `null` since it's a keyword and avoid starting a sentence with it. *Any* seems to work fine here.